index.js

```diff
@@ -1,21 +1,38 @@
 'use strict';
 
-module.exports = {
-  env: {
-    node: true,
-  },
-  extends: [
-    'crowdstrike',
-    'plugin:n/recommended',
-  ],
-  plugins: [
-    'json-files',
-  ],
-  rules: {
-    'strict': 'error',
+const jsonFiles = require('eslint-plugin-json-files');
+const globals = require('globals');
+const { FlatCompat } = require('@eslint/eslintrc');
+const js = require('@eslint/js');
+const nodePlugin = require('eslint-plugin-n');
 
-    'json-files/require-engines': 'error',
-    'json-files/require-license': 'error',
-    'json-files/restrict-ranges': ['error', { pinUnstable: true }],
-  },
-};
+const compat = new FlatCompat({
+  baseDirectory: __dirname, // optional; default: process.cwd()
+  resolvePluginsRelativeTo: __dirname, // optional
+  recommendedConfig: js.configs.recommended, // optional unless using "eslint:recommended"
+});
+
+module.exports = [
+  ...compat.extends('crowdstrike'),
+  nodePlugin.configs['flat/recommended-script'],
```

Comment:
Looks like this was added in https://github.com/eslint-community/eslint-plugin-n/releases/tag/16.1.0, so we'll need to bump the peer dep.